src/web/handlers/FlingGestureHandler.ts

```diff
@@ -114,6 +114,8 @@ export default class FlingGestureHandler extends GestureHandler {
 
     super.onPointerDown(event);
     this.newPointerAction();
+
+    this.tryToSendTouchEvent(event);
```

Comment:
Why not `super.onPointerDown(event)`?

Reply:
Because we don't send touch events in parent class anymore 😅

Reply:
Also, there is `super.onPointerDown(event)` visible on that suggestion (line 115) 😅